utils/lang.py

```diff
@@ -187,7 +187,7 @@ def cmd_generate_hash_tables(args):
     if not translations:
         logger.error('no translations found')
         return 1
-    buckets_count = 256
```

Comment:
Actually, do we want this?

This is the number of buckets we start checking for no collisions. Having it lower shouldn't break anything, it will just take a bit longer to generate the tables. 

So, in other words, are we sure that a non-colliding table will never have less than 340 buckets?

Reply:
Actually, I was thinking about the same - the less buckets we achieve the better. Right now the optimal number of buckets is 266, which is way less than 340 (which was chosen as a rule of thumb).

Reply:
And I didn't notice that your last commit changed it back to 256 :)

Nevertheless, based on our discussion on Slack, I changed it to 128 (commit c7ffcf3).